packages/dso-toolkit/src/components/selectable/selectable.scss

```diff
@@ -3,6 +3,8 @@
 
 // HTML/CSS
 .dso-selectable {
+  display: var(--dso-selectable-display, block);
```

Comment:
Volgens mij kan deze styling in de `@mixin selectable.root()`, kan je daar nog eens naar kijken?

Het is maar een klein dingetje, maar dit sluit perfect aan bij waar we het vanmiddag over hebben gehad.

Reply:
die wordt zo te zien in het webcomponent niet gebruikt, alleen in de html/css versie:
![Scherm­afbeelding 2023-10-12 om 08 36 25](https://github.com/dso-toolkit/dso-toolkit/assets/506527/bea02068-5bf3-4f1e-a34a-0de03735b92c)

Reply:
done!

Reply:
🎉